org.mwc.cmap.tests/.project

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@saulhidalgoaular - we don't want this file in the repo. You should delete the project from your Eclipse workspace.

Reply:
ohhh... ok. Sorry. I am deleting it!!